TTS/bin/extract_tts_spectrograms.py

```diff
@@ -229,7 +229,7 @@ def main(args):  # pylint: disable=redefined-outer-name
     ap = AudioProcessor(**c.audio)
 
     # load data instances
-    meta_data_train, meta_data_eval = load_tts_samples(c.datasets, eval_split=args.eval)
+    meta_data_train, meta_data_eval = load_tts_samples(c.datasets, eval_split=args.eval, eval_split_max_size=c.eval_split_max_size, eval_split_proportion=c.eval_split_proportion)
```

Comment:
you could go for a single argument and take it as ```size``` if it is int and ```portion``` if it is float. 

What do you think?

Reply:
I don't know, the idea of pass eval_split_max_size is to deal with big datasets without the user's needing to set a small value for portion.  Maybe we can keep the eval_split_max_size and just replace eval_split_proportion to eval_split_size and if it is int it is the number of samples and if it is floating the portion. What do you think?

Reply:
Makes sense but then eval_split_max_size should default to None and be discarded when it is None. Otherwise, without knowing users can think they split the dataset with the split portion but actually it picks only 500 samples.

Reply:
Done